react/CloseButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {FC} from 'react';
```

Comment:
Could you please lint this file? There are `;`s, `"`s mixed with `'`s and other details

Reply:
Just run `yarn` on the root of this repo, and also inside the `/react` folder. Then you should be able to run `yarn lint --fix`